### PR TITLE
Mina g

### PR DESCRIPTION
**Pull request by mghaffar89**
added footer with styling and the contact icons we want to use

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -117,6 +117,19 @@ h2 {
   margin-top: 1rem;
   margin-bottom: 1rem;
 }
+
+#footer {
```
Comment (mcraepv) [inline style.css:121]: you're adding fixed pixel values to it, which is going to break responsiveness. replace this with responsive units to fix those issues

File: about.html [hunk 2]
```diff
@@ -35,8 +35,8 @@
         <ul class="menu">
           <li><a href="searchcriteria.html">Search Librar.io</a></li>
           <li><a href="index.html">Book Journal</a></li>
+          <li><a href="saved.html">Saved Books</a></li>
```
Comment (mcraepv) [inline about.html:38]: I don't think we need another page for saved searches. It can be done on the book search page

File: style.css (hunk 1 above)
Comment (mcraepv) [inline style.css:121]: you're adding fixed pixel values to it, which is going to break responsiveness. replace this with responsive units to fix those issues

File: savedbooks.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
+var savedBooks = JSON.parse(localstorage.getItem("savedBooks"));
```
Comment (mcraepv) [inline savedbooks.js]: this can also be done on our existing files with firebase

File: style.css (hunk 1 above)
Comment (mghaffar89) [reply style.css:121]: I didnt add those, I think you put that in for the reset search button

File: about.html (hunk 2 above)
Comment (mghaffar89) [reply about.html:38]: Dont you think we need another page to display the saved books?

File: savedbooks.js (hunk 3 above)
Comment (mghaffar89) [reply savedbooks.js]: yeah i realized that after when I messaged you on slack that we are using firebase

**Review by mcraepv — APPROVED** [review]

**Pull request merged**